transforms/code/license_select/kfp_ray/license_select_wf.py

```diff
@@ -0,0 +1,211 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# NOTE: This file is auto generated by Pipeline Generator.
+import os
+
+import kfp.compiler as compiler
+import kfp.components as comp
+import kfp.dsl as dsl
+from workflow_support.compile_utils import ONE_HOUR_SEC, ONE_WEEK_SEC, ComponentUtils
+
+
+# the name of the job script
+EXEC_SCRIPT_NAME: str = "license_select_transform_ray.py"
+PREFIX: str = ""
+
+task_image = "quay.io/dataprep1/data-prep-kit/license_select-ray:0.4.0.dev6"
+
+# components
+base_kfp_image = "quay.io/dataprep1/data-prep-kit/kfp-data-processing:0.2.0.dev6"
```

Comment:
and this should be 0.2.1.dev0.  you might try running `make set-versions` from ..

Reply:
oops this should also be `latest`